raiden_contracts/data/deployment_goerli.json

```diff
@@ -16,8 +16,7 @@
       "gas_cost": 4579912,
       "constructor_arguments": [
         "0xDEF219a11e3d7485075704Ec9ade6eF994c0b5B7",
-        20,
-        555428,
+        200,
```

Comment:
Why did you change these? These files are written when depolying the contracts, and shouldn't be changed by hand.

Reply:
I couldn't find where these values were being set, I assumed they should be manually altered in the deployment files.

Reply:
No worries, we can sit together and deploy the contracts on goerli when we're both in the office. Just to see how the process looks like and when those files are created.